config/prod/nf-syn23664726-s3.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.8/templates/S3/synapse-external-bucket.j2"
+stack_name: "nf-syn23664726-s3"
+stack_tags:
+  Department: "SCCE"
+  Project: "neurofibromatosis"
+  OwnerEmail: 'robert.allaway@sagebionetworks.org'
+  CostCenter: "NTAP NF Addendum 4 / 301100"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:sts::237179673806:assumed-role/AWSReservedSSO_Community-Manager_84cea7c897f3f843/robert.allaway@sagebase.org'
```

Comment:
this is not going to work because the community manager role do not have access to this bucket.  do you have an IAM account anywhere else?  if not you could just setup a free aws account and create yourself an IAM user or role with bucket access.

Reply:
@zaro0508 , would it work to use my TowerViewer role for the nextflow prod account (AWSReservedSSO_TowerViewer_fd6aee98ec127705/robert.allaway@sagebase.org)? I really would prefer not to manage another AWS account.

Reply:
the [TowerViewer group policy definition](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L590-L621) say any user in that group *only* has access to view batch jobs and that's it.

Reply:
@allaway i have restored your access to sandbox to unblock you for now however please keep in mind that this may change.  You can use `arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/robert.allaway@sagebase.org` to access the bucket thru your account in sandbox.